.chronus/changes/fix_string_formats-2024-1-12-16-54-4.md

```diff
@@ -0,0 +1,7 @@
+---
+changeKind: fix
+packages:
+  - "@azure-tools/typespec-client-generator-core"
+---
+
+allows us to generate arm packages
```

Comment:
not sure this will make much sense to what is fixed here to someone looking at the changelog